feat: machine autoscaler documentation (#2754)
codenvy/che-docs: modules/administration-guide/pages/configuring-autoscaling.adoc

:_content-type: ASSEMBLY
:description: Configuring autoscaling
:keywords: administration-guide, configuring, autoscaling, autoscale, horizontal, scaling, machine
:navtitle: Configuring autoscaling
:page-aliases:

[id="configuring-autoscaling"]
= Configuring autoscaling

> This section describes different aspects of autoscaling for {prod}.

Reviewer: ```suggestion
Learn about different aspects of autoscaling for {prod}.
```

> This section describes different aspects of autoscaling for {prod}.

Reviewer: which docs do you want to link here? configuring machine autoscaling and configuring number of replicas?

Author: Yes, I've added them in my latest commit
